store/cache.go

```diff
@@ -85,6 +86,7 @@ func (cm *CacheManager) Get(cacheKey, targetPath string) error {
 	if err := os.Link(cm.path(cacheKey), targetPath); err != nil {
 		return err
 	}
+	logger.Debugf("CacheManager: using cache for %s", targetPath)
```

Comment:
The type name prefix sounds unnecessary and not very helpful for us or for users. Improving the message when unclear sounds better, but this message looks good on itself already.